src/database/models/notifications.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+module.exports = (Sequelize, DataTypes) => {
+  const Notifications = Sequelize.define('Notifications', {
+    modelId: { type: DataTypes.INTEGER, allowNull: false },
+    modelName: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      validate: {
+        isIn: {
+          args: [['Requests', 'Comments', 'Trips', 'Accommodations', 'AccommodationImages', 'Ratings', 'Likes']],
+          msg: 'That table doesn\'t exist',
+        },
+      },
+    },
+    type: { type: DataTypes.STRING, allowNull: false },
```

Comment:
- We would need a list of `isIn.args` here. Each `type` should be `snake_case`
- `['new_request', 'request_approved', 'request_rejected']`